jack.py

```diff
@@ -187,6 +187,35 @@
 void jack_set_info_function(void (*func)(const char*));
 void jack_free(void* ptr);
 
+/* ringbuffer.h */
+typedef struct {
+    char *buf;
```

Comment:
In the rest of the `cdef()` I changed the style to Stroustrup's C++ style (the asterisk is part of the type, not part of the variable).
Can you please also change this?

The style in the JACK headers is quite inconsistent, so I took the liberty of changing the style.